demos/mixer-config-quota-bookinfo.yaml

```diff
@@ -22,6 +22,50 @@ data:
         params:
       - name: default
         impl: denyChecker
+    attributes:
+      - name: source.name
+        value_type: 1 # STRING
+      - name: target.name
+        value_type: 1 # STRING
+      - name: api.name
```

Comment:
There is no API understanding in the proxy. "api.name" is probably referring to the API name in the Google's service config, which is separate from the service. A service can have multiple  APIs.